README.md

```diff
@@ -1,11 +1,11 @@
 # The TUF website
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/5c5979e1-541a-442a-aa4f-50abbd6c0b49/deploy-status)](https://app.netlify.com/sites/tuf-homepage/deploys)
+[![Netlify Status](https://api.netlify.com/api/v1/badges/91c8d69b-9b02-4c3a-ba58-100d65699e38/deploy-status)](https://app.netlify.com/sites/tufresources/deploys)
 
 Website repository for The Update Framework (TUF), build with [Hugo][] using the
 [Docsy][] theme, hosted on [Netlify][].
 
-Preview for the development version: https://tuf-homepage.netlify.app/
+Preview for the development version: https://tufresources.netlify.app/
```

Comment:
Could you set this to the main URL for this repo, rather than change it for each PR?

The PR's preview is available from the PR comments, see https://github.com/Dindihub/docsy-tuf.io/pull/12#issuecomment-2278346323, so it doesn't need to be added to the PR.

The same comment applies for line 3.

Reply:
Initially, I was having trouble setting it up for the main branch. When I did - https://tufio.netlify.app/ for some reason Netlify created one for the resources branch when I pushed. I'll take a look again. Thanks